builtins/target-neon-common.ll

```diff
@@ -197,6 +197,20 @@ define void @__fastmath() nounwind alwaysinline {
   ret void
 }
 
+define i32 @__set_ftz_daz_flags() nounwind alwaysinline {
```

Comment:
NEON 32 bit and NEON 64 bit seem to have different instructions / behavior. I suggest double checking that this works on both as expected.